### PR TITLE
documentation(EJ2-63146): Audio and Video Docs added for the aspcore and mvc.

### PR DESCRIPTION
**Pull request by IndrajithSrinivasan**
documentation(EJ2-63146): Audio and Video Docs added for the aspcore and mvc.

### CODE REVIEW

File: ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/rename.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+using System.IO;
+using System.Web;
+using System.Web.Mvc;
+
+namespace RenameImage.Controllers
+{
+    public class HomeController : Controller
+    {
+        int x = 0;
+        string imageFile;
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/rename.cs:11]: Why you have used the image word here?

File: ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/tagHelper [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<ejs-richtexteditor id="editor" imageUploadSuccess="onImageUploadSuccess">
+    <e-richtexteditor-insertimagesettings saveUrl="/Home/Rename" path="./Uploads/"></e-richtexteditor-insertimagesettings>
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/tagHelper:2]: Why you have used the insert image settings?

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/audio.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+documentation: ug
+---
+
+# Insert Audio
+
+The Rich Text Editor allows inserting audio files from online sources and the local computer where you want to insert the audio in your content. For inserting the audio to the Rich Text Editor, the following list of options have been provided in the [InsertAudioSettings](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditor.html#Syncfusion_EJ2_RichTextEditor_RichTextEditor_InsertAudioSettings).
+
+| Options | Description |
+|----------------|---------|
+| allowedTypes | Specifies the extensions of the audio types allowed to insert on bowering and passing the extensions with comma separators. For example, pass allowedTypes as `.mp3`, `.wav`, `.m4a` and `.wma` |
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/audio.md:17]: allowedTypes - use the Pascale casing in MVC and core. Check all other properties

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+| MaxHeight | Sets the maxHeight of the video element when it is inserted in the Rich Text Editor.|
+| SaveUrl | Provides URL to map the action result method to save the video.|
+| RemoveUrl | Provides URL to map the action result method to remove the video.|
+| Path | Specifies the location to store the video.|
+| Resize | Sets the resizing action for the video element.|
+| ResizeByPercent | Sets the percentage values for the video element with the resizing action.|
+
+## Configure video tool in the toolbar
+
+To include the video tool in the Rich Text Editor, you can add the toolbar item `Video` to the `toolbarSettings` [Items](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditorToolbarSettings.html#Syncfusion_EJ2_RichTextEditor_RichTextEditorToolbarSettings_Items) property.
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md:34]: Property casing

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+{% endtabs %}
+{% endif %}
+
+### Server-side action
+
+The selected video can be uploaded to the required destination using the controller action below. Map this method name in [InsertVideoSettings.SaveUrl](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditorVideoSettings.html#Syncfusion_EJ2_RichTextEditor_RichTextEditorVideoSettings_SaveUrl) and provide required destination path through [InsertVideoSettings.Path](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditorVideoSettings.html#Syncfusion_EJ2_RichTextEditor_RichTextEditorVideoSettings_Path) properties.
+
+> If you want to insert lower-sized video files in the editor and don't want a specific physical location for saving the video, you can save the format as `Base64`.
+
+In the following code blocks, the video module has been injected and can insert the video files saved in the specified path.
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md:118]: Don't use the inject module concept in MVC, core

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+documentation: ug
+---
+
+# Insert Audio
+
+The Rich Text Editor allows inserting audio files from online sources and the local computer where you want to insert the audio in your content. For inserting the audio to the Rich Text Editor, the following list of options have been provided in the [insertAudioSettings](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditor.html#Syncfusion_EJ2_RichTextEditor_RichTextEditor_InsertAudioSettings).
+
+| Options | Description |
+|----------------|---------|
+| allowedTypes | Specifies the extensions of the audio types allowed to insert on bowering and passing the extensions with comma separators. For example, pass allowedTypes as `.mp3`, `.wav`, `.m4a` and `.wma` |
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md:17]: Casing. Missing

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+| LayoutOption |Sets the default display for audio when it is inserted into the Rich Text Editor. Possible options are: `Inline` and `Break`.|
+| SaveFormat | Sets the default save format of the audio element when inserted. Possible options are: `Blob` and `Base64`.|
+| SaveUrl | Provides URL to map the action result method to save the audio.|
+| RemoveUrl | Provides URL to map the action result method to remove the audio.|
+| Path | Specifies the location to store the audio.|
+
+## Configure audio tool in the toolbar
+
+To include the audio tool in the Rich Text Editor, you can add the toolbar item `Audio` to the `toolbarSettings` [items](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditorToolbarSettings.html#Syncfusion_EJ2_RichTextEditor_RichTextEditorToolbarSettings_Items) property.
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md:26]: Property casing missing

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/video.md [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+{% endhighlight %}
+{% highlight c# tabtitle="SaveFile.cs" %}
+{% include code-snippet/rich-text-editor/video-save/saveFile.cs %}
+{% endhighlight %}
+{% endtabs %}
+{% endif %}
+
+### Video save format
+
+The video files can be saved as `Blob` or `Base64` url by using the [insertVideoSettings.saveFormat](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.RichTextEditor.RichTextEditorVideoSettings.html#Syncfusion_EJ2_RichTextEditor_RichTextEditorVideoSettings_SaveFormat) property, which is of enum type and the generated url will be set to the `src` attribute of the `<source>` tag.
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/video.md:145]: In core, ensure the breadcrumb component for property casing

File: ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/rename.cs (hunk 1 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/rename.cs:11]: Changed @thangavele

File: ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/tagHelper (hunk 2 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/code-snippet/rich-text-editor/audio-rename/tagHelper:2]: Changed @thangavele

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/audio.md (hunk 3 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/audio.md:17]: Changed in MVC, @thangavele 

MVC alone pascal casing occurs for properties. In core the casing is similar to TypeScript ways.

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md (hunk 4 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md:34]: okay @thangavele

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md (hunk 5 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/video.md:118]: okay @thangavele

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/video.md (hunk 8 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/video.md:145]: Breadcrumb used only small casing for core platform. @thangavele

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md (hunk 7 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md:26]: changed for MVC, for core this is correct @thangavele

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md (hunk 6 above)
Comment (IndrajithSrinivasan) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/audio.md:17]: changed for MVC, for core this is correct @thangavele

**Review by IndrajithSrinivasan — COMMENTED** [review]
Updated my comments

**Review by saravanangn — APPROVED** [review]

**Pull request merged**